vllm/model_executor/layers/quantization/gptq_marlin.py

```diff
@@ -46,11 +42,18 @@ def _get_perms():
             ]:
                 perm1.append(16 * row + col + 8 * block)
         for j in range(4):
-            perm.extend([p + 256 * j for p in perm1])
+            perm_list.extend([p + 256 * j for p in perm1])
+
+    perm = numpy.array(perm_list)
+
+    if num_bits == 4:
+        interleave = numpy.array([0, 2, 4, 6, 1, 3, 5, 7])
+    elif num_bits == 8:
+        interleave = numpy.array([0, 2, 1, 3])
+    else:
+        raise Exception("num_bits must be 4 or 8, got {}".format(num_bits))
```

Comment:
ValueError? Raising generic exceptions is not a good idea :)

Reply:
Removed